dll/win32/shell32/shfldr.h

```diff
@@ -49,6 +49,8 @@ For Details See:
 
 #define GET_SHGDN_FOR(dwFlags)         ((DWORD)dwFlags & (DWORD)0x0000FF00)
 #define GET_SHGDN_RELATION(dwFlags)    ((DWORD)dwFlags & (DWORD)0x000000FF)
+#define IS_SHGDN_FOR_PARSING(flags) ( ((flags) & (SHGDN_FORADDRESSBAR | SHGDN_FORPARSING)) == SHGDN_FORPARSING)
```

Comment:
please indent (on tab-aligned spaces) the definitions of these macros to ease reading (as done for the other defines in the file)

Reply:
I'd suggest that you fix our .clang-format to do this if this is so important?